crates/sui-kvstore/src/bigtable/proto/google.bigtable.v2.rs

```diff
@@ -0,0 +1,1734 @@
+/// Specifies the complete (requested) contents of a single row of a table.
+/// Rows which exceed 256MiB in size cannot be read in full.
+#[derive(Clone, PartialEq, ::prost::Message)]
+pub struct Row {
```

Comment:
How did you generate this code? I'm assuming with prost-build? Would it be possible to include those details somewhere?